app/assets/stylesheets/pages/_home.scss

```diff
@@ -8,7 +8,8 @@
 }
 
 .homepage-container {
-  padding-top: 160px;
+  padding-top: 20px;
```

Comment:
this should be 
    padding-top: 160px;
    position: absolute;
    top: 0;
    bottom: 0;
    left: 0;
    right: 0;